### ISSUE TITLE
[Issue]: sees my phone as googled after last update

### ISSUE THREAD

**Issue by Martinligabue** [techlore/Plexus-app#46]
### Description

After last update, 2.0.7 I can't submit a review

### Steps to reproduce the issue

Open app page
Tap on send review
_Device must be degoogled or microg..._ 

### Device

microG

### Additional context

Tell me what I can do to provide additional info

### Terms

- [X] This issue occurs on the [latest version](https://github.com/techlore/Plexus-app/releases/latest).
- [X] This is not a duplicate of another issue.

**Comment by StellarSand** [comment]
That shouldn't be happening. I tested on a microG device & I don't see that snackbar message.

[1.webm](https://github.com/user-attachments/assets/b771c493-146b-494e-85de-bfd57c66840f) [2.webm](https://github.com/user-attachments/assets/85986c68-b605-4ef7-a88e-ad9c1d1c3c19)

Could you please verify:
- if any of the following packages are installed on your device:
    - `com.google.android.gms`
    - `com.google.android.gsf`
    - `com.android.vending`
- the above packages are the microG versions and not the standard Google ones.

**Comment by sbarnard86** [comment]
This is also happening on my degoogled device. I'm running GrapheneOS on a Pixel 7 and everything is up to date.

**Comment by StellarSand** [comment]
@sbarnard86 please verify if any of the following packages are installed on your device:
- com.google.android.gms
 - com.google.android.gsf
 - com.android.vending

**Comment by sbarnard86** [comment]
@StellarSand I am seeing gms and vending but not gsf

**Comment by StellarSand** [comment]
@sbarnard86 

- `com.google.android.gms` is `Google Play services`/`microG Services`.
- `com.android.vending` is `Google Play Store`/`microG Companion`.

If the microG services were installed, then the app wouldn't show you the snackbar saying "Device must be degoogled or microG". So your device has the google services installed.
GrapheneOS is degoogled so the google services shouldn't even be installed in the first place, unless you are using sandboxed google services? To the best of my knowledge, this feature runs google services in a sandboxed environment, separate from the main profile, but I could be mistaken.

**Comment by sbarnard86** [comment]
@StellarSand Yeah I have sandboxed Google Play running. I just have it going for Google maps. So if I turn it off/uninstall it, I should be able to use Plexus?

**Comment by StellarSand** [comment]
@sbarnard86 yes. Alternatively you can set up a separate profile without google services and install the app on that profile too.

**Comment by EdoSecco** [comment]
I'm experiencing similar trouble on a phone LineageOS without Google: after last update 30 Dec the app incorrectly sees it as MicroG and I can't make reviews for degoogled.

**Comment by StellarSand** [comment]
@EdoSecco please verify if any of the following packages is installed on your device:

- `com.google.android.gms`
- `com.google.android.gsf`
- `com.android.vending`

**Comment by EdoSecco** [comment]
No one of these; as said, it is LineageOS completely without G components, but I checked all the packages to be sure.
It is just strange that until few days ago with 2.0.7 all was fine, this led me thinking it's a bug in the last version.

**Comment by CristianKerr** [comment]
I have just installed the application and it's incorrectly detecting microG. I don't have microG in this profile (where I installed the app)/ I have them in work profile, but that shouldn't play any role...

**Comment by StellarSand** [comment]
@CristianKerr I'm aware this is an issue. Which ROM?

**Comment by CristianKerr** [comment]
@StellarSand Perfect. I am using CalyxOS. Thank you!

**Comment by StellarSand** [comment]
@CristianKerr Yes, it was fixed in [this commit](https://github.com/techlore/Plexus-app/commit/ca8e0041ceafbcb7ce3e7004c745388c3a10c151).

**Comment by CristianKerr** [comment]
@StellarSand  the fix doesn't work for me then

**Comment by StellarSand** [comment]
@CristianKerr Are you sure the microG apps are really on a separate profile and not on the one you have Plexus app installed?

**Comment by CristianKerr** [comment]
@StellarSand yes, nothing on my main profile, microG on my work profile.

**Comment by StellarSand** [comment]
@CristianKerr I tested on a deGoogled device with no microG on main profile, only on the work profile. 

Here's the results:
https://github.com/user-attachments/assets/c675465a-effb-4a16-8c9d-706c6524da45

As you can see, with v2.0.9 it was indeed incorrectly detecting the deGoogled device as microG, but after updating to v2.0.1, it correctly identifies as deGoogled.

**Comment by CristianKerr** [comment]
@StellarSand Then there is probably another bug. Why can't the users change the setting anyway? You can pre-set it but if it's wrong user might just change it. 

https://github.com/user-attachments/assets/9ee0e38c-f6cc-486e-9a5b-e52c45dd4c1c

**Comment by StellarSand** [comment]
@CristianKerr Hmm that's weird 🤔 

> Why can't the users change the setting anyway?

Because then users can select "deGoogled device" on a microG device or vice versa whenever they want and all those ratings will have flawed data.

**Comment by lucasmz-dev** [comment]
Can reproduce. CalyxOS 5.14.2, rhode.

**Comment by lucasmz-dev** [comment]
I don't see any microG components when doing it, via Settings > Apps (with "Show system apps")...

**Comment by lucasmz-dev** [comment]
Huh. I *can* see it using APK Explorer & Editor, all of them. Weird.

**Comment by brezelman** [comment]
related: https://github.com/techlore/plexus/issues/378
